### PR TITLE
Fai 71 vacancies manage api 

### PR DESCRIPTION
**Pull request by A-J-Wright**

### CODE REVIEW

**Review by dashton82 — DISMISSED** [review]

File: deployments/vacanciesmanage.yml [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+- name: OuterApiObject
+  value: $(VacanciesManageOuterApiObject)
+
+resources:
+  repositories:
+  - repository: self
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/tags/0.3.31
```
Comment (jack-cloud-platform) [inline deployments/vacanciesmanage.yml:16]:
```suggestion
    ref: refs/tags/0.4.10
```
Can be latest tag

File: deployments/vacanciesmanage.yml [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    endpoint: SkillsFundingAgency
+  - repository: das-platform-automation
+    type: github
+    name: SkillsFundingAgency/das-platform-automation
+    ref: refs/tags/4.4.1
```
Comment (jack-cloud-platform) [inline deployments/vacanciesmanage.yml:21]:
```suggestion
    ref: refs/tags/4.5.0
```
Can be latest tag

Comment (jack-cloud-platform) [reply deployments/vacanciesmanage.yml:21]: https://github.com/SkillsFundingAgency/das-platform-automation/releases

Comment (dashton82) [reply deployments/vacanciesmanage.yml]: 👍🏻 thats done

**Pull request merged**